Error installing neo4j

### ✅ Have you read and understood the above guidelines?

yes

### 📜 What is the name of the script you are using?

neo4j

### 📂 What was the exact command used to execute the script?

bash -c "$(curl -fsSL https://raw.githubusercontent.com/community-scripts/ProxmoxVE/main/ct/neo4j.sh)"

### 📝 Provide a clear and concise description of the issue.

Upon running the above script the followin error is encountered
```
The following packages have unmet dependencies:
 neo4j : Depends: java21-runtime but it is not installable or
                  jdk-21 but it is not installable
         Depends: cypher-shell (>= 1:2025.03.0) but it is not going to be installed
         Depends: cypher-shell (< 2:0.0.0) but it is not going to be installed
E: Unable to correct problems, you have held broken packages.
 ⠏
[ERROR] in line 25: exit code 0: while executing command $STD apt-get install -y neo4j
```

### ⚙️ What settings are you using?

- [x] Default Settings
- [ ] Advanced Settings

### 🖥️ Which Linux distribution are you using?

Debian 12

### 🔄 Steps to reproduce the issue.
Step 1: Login to proxmox and go to built-in console
Step 2: Paste command: `bash -c "$(curl -fsSL https://raw.githubusercontent.com/community-scripts/ProxmoxVE/main/ct/neo4j.sh)"`
Step 3: Select default settings and proceed with install


### ❌ Paste the full error output (if available).

    _   __           __ __  _ 
   / | / /__  ____  / // / (_)
  /  |/ / _ \/ __ \/ // /_/ / 
 / /|  /  __/ /_/ /__  __/ /  
/_/ |_/\___/\____/  /_/_/ /   
                     /___/    
  ⚙️  Using Default Settings on node pve (🔍  Verbose)
  🖥️  Operating System: debian
  🌟  Version: 12
  📦  Container Type: Unprivileged
  💾  Disk Size: 4 GB
  🧠  CPU Cores: 1
  🛠️  RAM Size: 1024 MiB
  🆔  Container ID: 119
  🔍  Verbose Mode: Enabled
  🚀  Creating a Neo4j LXC using the above default settings
  
  ✔️  Using local for Template Storage.
  ✔️  Using local-zfs for Container Storage.
  ✔️  Updated LXC Template List
  ✔️  LXC Template is ready to use.
  ✔️  LXC Container 119 was successfully created.
  ✔️   Started LXC Container
  ✔️  Set up Container OS
  ✔️  Network Connected: 192.168.10.175 
  ✔️  IPv4 Internet Connected
  ✖️  IPv6 Internet Not Connected
  ✔️  DNS Resolved github.com to 20.207.73.82
Get:1 http://deb.debian.org/debian bookworm InRelease [151 kB]
Get:2 http://security.debian.org bookworm-security InRelease [48.0 kB]
Get:3 http://deb.debian.org/debian bookworm-updates InRelease [55.4 kB]
Get:4 http://security.debian.org bookworm-security/main amd64 Packages [254 kB]
Get:5 http://security.debian.org bookworm-security/main Translation-en [150 kB]
Get:6 http://security.debian.org bookworm-security/contrib amd64 Packages [896 B]
Get:7 http://security.debian.org bookworm-security/contrib Translation-en [652 B]
Get:8 http://deb.debian.org/debian bookworm/main amd64 Packages [8,792 kB]
Get:9 http://deb.debian.org/debian bookworm-updates/main amd64 Packages.diff/Index [16.2 kB]
Get:10 http://deb.debian.org/debian bookworm/main Translation-en [6,109 kB]
Get:11 http://deb.debian.org/debian bookworm/contrib amd64 Packages [54.1 kB]
Get:12 http://deb.debian.org/debian bookworm/contrib Translation-en [48.8 kB]
Get:13 http://deb.debian.org/debian bookworm-updates/main Translation-en [360 B]
Get:14 http://deb.debian.org/debian bookworm-updates/main amd64 Packages T-2025-03-27-1414.15-F-2024-11-27-1405.46.pdiff [435 B]
Get:14 http://deb.debian.org/debian bookworm-updates/main amd64 Packages T-2025-03-27-1414.15-F-2024-11-27-1405.46.pdiff [435 B]
Fetched 15.7 MB in 1s (13.7 MB/s)                
Reading package lists... Done
N: Repository 'http://deb.debian.org/debian bookworm InRelease' changed its 'Version' value from '12.7' to '12.10'
Reading package lists... Done
Building dependency tree... Done
Calculating upgrade... Done
The following packages will be upgraded:
  base-files bind9-dnsutils bind9-host bind9-libs bsdextrautils bsdutils distro-info-data fdisk iputils-ping libblkid1 libc-bin libc-l10n libc6 libcurl3-gnutls
  libexpat1 libfdisk1 libgnutls30 liblzma5 libmount1 libnghttp2-14 libnss-systemd libpam-systemd libperl5.36 libpython3.11-minimal libpython3.11-stdlib
  libsmartcols1 libsqlite3-0 libssl3 libsystemd-shared libsystemd0 libtasn1-6 libudev1 libuuid1 locales mount openssh-client openssh-server openssh-sftp-server
 ⠧  openssl perl perl-base perl-modules-5.36 python3-pkg-resources python3-urllib3 python3.11 python3.11-minimal ssh systemd systemd-sysv systemd-timesyncd tzdata
  ucf udev util-linux util-linux-extra vim-common vim-tiny wget xz-utils
59 upgraded, 0 newly installed, 0 to remove and 0 not upgraded.
Need to get 44.6 MB of archives.
After this operation, 603 kB disk space will be freed.
Get:1 http://deb.debian.org/debian bookworm/main amd64 base-files amd64 12.4+deb12u10 [70.9 kB]
Get:2 http://security.debian.org bookworm-security/main amd64 libperl5.36 amd64 5.36.0-7+deb12u2 [4,207 kB]
Get:3 http://deb.debian.org/debian bookworm/main amd64 bsdutils amd64 1:2.38.1-5+deb12u3 [94.4 kB]
Get:4 http://deb.debian.org/debian bookworm/main amd64 libc6 amd64 2.36-9+deb12u10 [2,757 kB]
Get:5 http://deb.debian.org/debian bookworm/main amd64 libsmartcols1 amd64 2.38.1-5+deb12u3 [107 kB]
Get:6 http://deb.debian.org/debian bookworm/main amd64 util-linux-extra amd64 2.38.1-5+deb12u3 [111 kB]
Get:7 http://deb.debian.org/debian bookworm/main amd64 util-linux amd64 2.38.1-5+deb12u3 [1,177 kB]
Get:8 http://deb.debian.org/debian bookworm/main amd64 libc-bin amd64 2.36-9+deb12u10 [608 kB]
Get:9 http://deb.debian.org/debian bookworm/main amd64 mount amd64 2.38.1-5+deb12u3 [134 kB]
Get:10 http://deb.debian.org/debian bookworm/main amd64 libnss-systemd amd64 252.36-1~deb12u1 [164 kB]
Get:11 http://deb.debian.org/debian bookworm/main amd64 libblkid1 amd64 2.38.1-5+deb12u3 [147 kB]
Get:12 http://deb.debian.org/debian bookworm/main amd64 libmount1 amd64 2.38.1-5+deb12u3 [166 kB]
Get:13 http://deb.debian.org/debian bookworm/main amd64 libssl3 amd64 3.0.15-1~deb12u1 [2,025 kB]
Get:14 http://deb.debian.org/debian bookworm/main amd64 systemd-timesyncd amd64 252.36-1~deb12u1 [63.7 kB]
Get:15 http://deb.debian.org/debian bookworm/main amd64 libpam-systemd amd64 252.36-1~deb12u1 [225 kB]
Get:16 http://deb.debian.org/debian bookworm/main amd64 systemd amd64 252.36-1~deb12u1 [3,037 kB]
Get:17 http://security.debian.org bookworm-security/main amd64 perl amd64 5.36.0-7+deb12u2 [239 kB]
Get:18 http://security.debian.org bookworm-security/main amd64 perl-base amd64 5.36.0-7+deb12u2 [1,609 kB]
Get:19 http://deb.debian.org/debian bookworm/main amd64 libsystemd-shared amd64 252.36-1~deb12u1 [1,693 kB]
Get:20 http://deb.debian.org/debian bookworm/main amd64 systemd-sysv amd64 252.36-1~deb12u1 [42.6 kB]
Get:21 http://deb.debian.org/debian bookworm/main amd64 libuuid1 amd64 2.38.1-5+deb12u3 [28.8 kB]
Get:22 http://deb.debian.org/debian bookworm/main amd64 libfdisk1 amd64 2.38.1-5+deb12u3 [194 kB]
Get:23 http://security.debian.org bookworm-security/main amd64 perl-modules-5.36 all 5.36.0-7+deb12u2 [2,815 kB]
Get:24 http://deb.debian.org/debian bookworm/main amd64 libsystemd0 amd64 252.36-1~deb12u1 [332 kB]
Get:25 http://deb.debian.org/debian bookworm/main amd64 openssh-sftp-server amd64 1:9.2p1-2+deb12u5 [65.8 kB]
Get:26 http://deb.debian.org/debian bookworm/main amd64 openssh-server amd64 1:9.2p1-2+deb12u5 [457 kB]
Get:27 http://deb.debian.org/debian bookworm/main amd64 openssh-client amd64 1:9.2p1-2+deb12u5 [991 kB]
Get:28 http://deb.debian.org/debian bookworm/main amd64 ucf all 3.0043+nmu1+deb12u1 [55.3 kB]
Get:29 http://deb.debian.org/debian bookworm/main amd64 python3.11 amd64 3.11.2-6+deb12u5 [573 kB]
Get:30 http://deb.debian.org/debian bookworm/main amd64 libpython3.11-stdlib amd64 3.11.2-6+deb12u5 [1,797 kB]
Get:31 http://deb.debian.org/debian bookworm/main amd64 python3.11-minimal amd64 3.11.2-6+deb12u5 [2,067 kB]
Get:32 http://security.debian.org bookworm-security/main amd64 liblzma5 amd64 5.4.1-1 [205 kB]
Get:33 http://security.debian.org bookworm-security/main amd64 xz-utils amd64 5.4.1-1 [471 kB]
Get:34 http://deb.debian.org/debian bookworm/main amd64 libpython3.11-minimal amd64 3.11.2-6+deb12u5 [816 kB]
Get:35 http://deb.debian.org/debian bookworm/main amd64 libsqlite3-0 amd64 3.40.1-2+deb12u1 [839 kB]
Get:36 http://deb.debian.org/debian bookworm/main amd64 libexpat1 amd64 2.5.0-1+deb12u1 [98.9 kB]
Get:37 http://deb.debian.org/debian bookworm/main amd64 libtasn1-6 amd64 4.19.0-2+deb12u1 [48.6 kB]
Get:38 http://deb.debian.org/debian bookworm/main amd64 libgnutls30 amd64 3.7.9-2+deb12u4 [1,405 kB]
Get:39 http://deb.debian.org/debian bookworm/main amd64 udev amd64 252.36-1~deb12u1 [1,704 kB]
Get:40 http://deb.debian.org/debian bookworm/main amd64 libudev1 amd64 252.36-1~deb12u1 [109 kB]
Get:41 http://deb.debian.org/debian bookworm-updates/main amd64 tzdata all 2025b-0+deb12u1 [299 kB]
Get:42 http://deb.debian.org/debian bookworm/main amd64 fdisk amd64 2.38.1-5+deb12u3 [141 kB]
Get:43 http://deb.debian.org/debian bookworm/main amd64 iputils-ping amd64 3:20221126-1+deb12u1 [47.2 kB]
Get:44 http://deb.debian.org/debian bookworm/main amd64 vim-tiny amd64 2:9.0.1378-2+deb12u2 [720 kB]
Get:45 http://deb.debian.org/debian bookworm/main amd64 vim-common all 2:9.0.1378-2+deb12u2 [125 kB]
Get:46 http://deb.debian.org/debian bookworm/main amd64 libnghttp2-14 amd64 1.52.0-1+deb12u2 [73.0 kB]
Get:47 http://deb.debian.org/debian bookworm/main amd64 bind9-dnsutils amd64 1:9.18.33-1~deb12u2 [155 kB]
Get:48 http://deb.debian.org/debian bookworm/main amd64 bind9-host amd64 1:9.18.33-1~deb12u2 [54.1 kB]
Get:49 http://deb.debian.org/debian bookworm/main amd64 bind9-libs amd64 1:9.18.33-1~deb12u2 [1,181 kB]
Get:50 http://deb.debian.org/debian bookworm/main amd64 libc-l10n all 2.36-9+deb12u10 [676 kB]
Get:51 http://deb.debian.org/debian bookworm/main amd64 locales all 2.36-9+deb12u10 [3,905 kB]
Get:52 http://deb.debian.org/debian bookworm/main amd64 wget amd64 1.21.3-1+deb12u1 [937 kB]
Get:53 http://deb.debian.org/debian bookworm/main amd64 bsdextrautils amd64 2.38.1-5+deb12u3 [87.0 kB]
Get:54 http://deb.debian.org/debian bookworm/main amd64 distro-info-data all 0.58+deb12u3 [6,272 B]
Get:55 http://deb.debian.org/debian bookworm/main amd64 libcurl3-gnutls amd64 7.88.1-10+deb12u12 [386 kB]
Get:56 http://deb.debian.org/debian bookworm/main amd64 openssl amd64 3.0.15-1~deb12u1 [1,427 kB]
Get:57 http://deb.debian.org/debian bookworm/main amd64 python3-pkg-resources all 66.1.1-1+deb12u1 [296 kB]
Get:58 http://deb.debian.org/debian bookworm/main amd64 python3-urllib3 all 1.26.12-1+deb12u1 [114 kB]
Get:59 http://deb.debian.org/debian bookworm/main amd64 ssh all 1:9.2p1-2+deb12u5 [174 kB]
Fetched 44.6 MB in 1s (60.9 MB/s)         
 ⠴apt-listchanges: Reading changelogs...
Extracting templates from packages: 100%
Preconfiguring packages ...
(Reading database ... 19138 files and directories currently installed.)
Preparing to unpack .../base-files_12.4+deb12u10_amd64.deb ...
Unpacking base-files (12.4+deb12u10) over (12.4+deb12u7) ...
Setting up base-files (12.4+deb12u10) ...
Installing new version of config file /etc/debian_version ...
(Reading database ... 19138 files and directories currently installed.)
Preparing to unpack .../bsdutils_1%3a2.38.1-5+deb12u3_amd64.deb ...
Unpacking bsdutils (1:2.38.1-5+deb12u3) over (1:2.38.1-5+deb12u1) ...
Setting up bsdutils (1:2.38.1-5+deb12u3) ...
(Reading database ... 19138 files and directories currently installed.)
Preparing to unpack .../libperl5.36_5.36.0-7+deb12u2_amd64.deb ...
Unpacking libperl5.36:amd64 (5.36.0-7+deb12u2) over (5.36.0-7+deb12u1) ...
 ⠇Preparing to unpack .../perl_5.36.0-7+deb12u2_amd64.deb ...
Unpacking perl (5.36.0-7+deb12u2) over (5.36.0-7+deb12u1) ...
Preparing to unpack .../perl-base_5.36.0-7+deb12u2_amd64.deb ...
Unpacking perl-base (5.36.0-7+deb12u2) over (5.36.0-7+deb12u1) ...
 ⠹Setting up perl-base (5.36.0-7+deb12u2) ...
(Reading database ... 19138 files and directories currently installed.)
Preparing to unpack .../perl-modules-5.36_5.36.0-7+deb12u2_all.deb ...
Unpacking perl-modules-5.36 (5.36.0-7+deb12u2) over (5.36.0-7+deb12u1) ...
 ⠏Preparing to unpack .../libc6_2.36-9+deb12u10_amd64.deb ...
Unpacking libc6:amd64 (2.36-9+deb12u10) over (2.36-9+deb12u8) ...
 ⠹Setting up libc6:amd64 (2.36-9+deb12u10) ...
(Reading database ... 19138 files and directories currently installed.)
Preparing to unpack .../libsmartcols1_2.38.1-5+deb12u3_amd64.deb ...
Unpacking libsmartcols1:amd64 (2.38.1-5+deb12u3) over (2.38.1-5+deb12u1) ...
 ⠴Setting up libsmartcols1:amd64 (2.38.1-5+deb12u3) ...
(Reading database ... 19138 files and directories currently installed.)
Preparing to unpack .../util-linux-extra_2.38.1-5+deb12u3_amd64.deb ...
Unpacking util-linux-extra (2.38.1-5+deb12u3) over (2.38.1-5+deb12u1) ...
Setting up util-linux-extra (2.38.1-5+deb12u3) ...
(Reading database ... 19138 files and directories currently installed.)
Preparing to unpack .../util-linux_2.38.1-5+deb12u3_amd64.deb ...
Unpacking util-linux (2.38.1-5+deb12u3) over (2.38.1-5+deb12u1) ...
 ⠏Setting up util-linux (2.38.1-5+deb12u3) ...
 ⠙fstrim.service is a disabled or a static unit not running, not starting it.
(Reading database ... 19138 files and directories currently installed.)
Preparing to unpack .../libc-bin_2.36-9+deb12u10_amd64.deb ...
Unpacking libc-bin (2.36-9+deb12u10) over (2.36-9+deb12u8) ...
Setting up libc-bin (2.36-9+deb12u10) ...
(Reading database ... 19138 files and directories currently installed.)
Preparing to unpack .../mount_2.38.1-5+deb12u3_amd64.deb ...
Unpacking mount (2.38.1-5+deb12u3) over (2.38.1-5+deb12u1) ...
Preparing to unpack .../libnss-systemd_252.36-1~deb12u1_amd64.deb ...
Unpacking libnss-systemd:amd64 (252.36-1~deb12u1) over (252.30-1~deb12u2) ...
Preparing to unpack .../libblkid1_2.38.1-5+deb12u3_amd64.deb ...
Unpacking libblkid1:amd64 (2.38.1-5+deb12u3) over (2.38.1-5+deb12u1) ...
 ⠼Setting up libblkid1:amd64 (2.38.1-5+deb12u3) ...
(Reading database ... 19138 files and directories currently installed.)
Preparing to unpack .../liblzma5_5.4.1-1_amd64.deb ...
Unpacking liblzma5:amd64 (5.4.1-1) over (5.4.1-0.2) ...
Setting up liblzma5:amd64 (5.4.1-1) ...
(Reading database ... 19138 files and directories currently installed.)
Preparing to unpack .../libmount1_2.38.1-5+deb12u3_amd64.deb ...
Unpacking libmount1:amd64 (2.38.1-5+deb12u3) over (2.38.1-5+deb12u1) ...
 ⠴Setting up libmount1:amd64 (2.38.1-5+deb12u3) ...
(Reading database ... 19138 files and directories currently installed.)
Preparing to unpack .../libssl3_3.0.15-1~deb12u1_amd64.deb ...
Unpacking libssl3:amd64 (3.0.15-1~deb12u1) over (3.0.14-1~deb12u2) ...
 ⠦Preparing to unpack .../systemd-timesyncd_252.36-1~deb12u1_amd64.deb ...
Unpacking systemd-timesyncd (252.36-1~deb12u1) over (252.30-1~deb12u2) ...
Preparing to unpack .../libpam-systemd_252.36-1~deb12u1_amd64.deb ...
Unpacking libpam-systemd:amd64 (252.36-1~deb12u1) over (252.30-1~deb12u2) ...
 ⠧Setting up libssl3:amd64 (3.0.15-1~deb12u1) ...
(Reading database ... 19138 files and directories currently installed.)
Preparing to unpack .../systemd_252.36-1~deb12u1_amd64.deb ...
Unpacking systemd (252.36-1~deb12u1) over (252.30-1~deb12u2) ...
 ⠙Preparing to unpack .../libsystemd-shared_252.36-1~deb12u1_amd64.deb ...
Unpacking libsystemd-shared:amd64 (252.36-1~deb12u1) over (252.30-1~deb12u2) ...
 ⠹Preparing to unpack .../libsystemd0_252.36-1~deb12u1_amd64.deb ...
Unpacking libsystemd0:amd64 (252.36-1~deb12u1) over (252.30-1~deb12u2) ...
 ⠸Setting up libsystemd0:amd64 (252.36-1~deb12u1) ...
Setting up libsystemd-shared:amd64 (252.36-1~deb12u1) ...
Setting up mount (2.38.1-5+deb12u3) ...
Setting up systemd (252.36-1~deb12u1) ...
(Reading database ... 19138 files and directories currently installed.)
Preparing to unpack .../systemd-sysv_252.36-1~deb12u1_amd64.deb ...
Unpacking systemd-sysv (252.36-1~deb12u1) over (252.30-1~deb12u2) ...
Preparing to unpack .../libuuid1_2.38.1-5+deb12u3_amd64.deb ...
Unpacking libuuid1:amd64 (2.38.1-5+deb12u3) over (2.38.1-5+deb12u1) ...
Setting up libuuid1:amd64 (2.38.1-5+deb12u3) ...
(Reading database ... 19138 files and directories currently installed.)
Preparing to unpack .../00-libfdisk1_2.38.1-5+deb12u3_amd64.deb ...
Unpacking libfdisk1:amd64 (2.38.1-5+deb12u3) over (2.38.1-5+deb12u1) ...
Preparing to unpack .../01-openssh-sftp-server_1%3a9.2p1-2+deb12u5_amd64.deb ...
Unpacking openssh-sftp-server (1:9.2p1-2+deb12u5) over (1:9.2p1-2+deb12u3) ...
Preparing to unpack .../02-openssh-server_1%3a9.2p1-2+deb12u5_amd64.deb ...
Unpacking openssh-server (1:9.2p1-2+deb12u5) over (1:9.2p1-2+deb12u3) ...
 ⠧Preparing to unpack .../03-openssh-client_1%3a9.2p1-2+deb12u5_amd64.deb ...
Unpacking openssh-client (1:9.2p1-2+deb12u5) over (1:9.2p1-2+deb12u3) ...
 ⠇Preparing to unpack .../04-ucf_3.0043+nmu1+deb12u1_all.deb ...
Unpacking ucf (3.0043+nmu1+deb12u1) over (3.0043+nmu1) ...
Preparing to unpack .../05-python3.11_3.11.2-6+deb12u5_amd64.deb ...
Unpacking python3.11 (3.11.2-6+deb12u5) over (3.11.2-6+deb12u3) ...
Preparing to unpack .../06-libpython3.11-stdlib_3.11.2-6+deb12u5_amd64.deb ...
 ⠏Unpacking libpython3.11-stdlib:amd64 (3.11.2-6+deb12u5) over (3.11.2-6+deb12u3) ...
 ⠙Preparing to unpack .../07-python3.11-minimal_3.11.2-6+deb12u5_amd64.deb ...
Unpacking python3.11-minimal (3.11.2-6+deb12u5) over (3.11.2-6+deb12u3) ...
 ⠹Preparing to unpack .../08-libpython3.11-minimal_3.11.2-6+deb12u5_amd64.deb ...
Unpacking libpython3.11-minimal:amd64 (3.11.2-6+deb12u5) over (3.11.2-6+deb12u3) ...
 ⠼Preparing to unpack .../09-libsqlite3-0_3.40.1-2+deb12u1_amd64.deb ...
Unpacking libsqlite3-0:amd64 (3.40.1-2+deb12u1) over (3.40.1-2) ...
 ⠴Preparing to unpack .../10-libexpat1_2.5.0-1+deb12u1_amd64.deb ...
Unpacking libexpat1:amd64 (2.5.0-1+deb12u1) over (2.5.0-1) ...
Preparing to unpack .../11-libtasn1-6_4.19.0-2+deb12u1_amd64.deb ...
Unpacking libtasn1-6:amd64 (4.19.0-2+deb12u1) over (4.19.0-2) ...
Setting up libtasn1-6:amd64 (4.19.0-2+deb12u1) ...
(Reading database ... 19138 files and directories currently installed.)
Preparing to unpack .../libgnutls30_3.7.9-2+deb12u4_amd64.deb ...
Unpacking libgnutls30:amd64 (3.7.9-2+deb12u4) over (3.7.9-2+deb12u3) ...
 ⠦Setting up libgnutls30:amd64 (3.7.9-2+deb12u4) ...
(Reading database ... 19138 files and directories currently installed.)
Preparing to unpack .../udev_252.36-1~deb12u1_amd64.deb ...
Unpacking udev (252.36-1~deb12u1) over (252.30-1~deb12u2) ...
 ⠇Preparing to unpack .../libudev1_252.36-1~deb12u1_amd64.deb ...
Unpacking libudev1:amd64 (252.36-1~deb12u1) over (252.30-1~deb12u2) ...
Setting up libudev1:amd64 (252.36-1~deb12u1) ...
(Reading database ... 19138 files and directories currently installed.)
Preparing to unpack .../00-tzdata_2025b-0+deb12u1_all.deb ...
 ⠏Unpacking tzdata (2025b-0+deb12u1) over (2024a-0+deb12u1) ...
 ⠸Preparing to unpack .../01-fdisk_2.38.1-5+deb12u3_amd64.deb ...
Unpacking fdisk (2.38.1-5+deb12u3) over (2.38.1-5+deb12u1) ...
Preparing to unpack .../02-iputils-ping_3%3a20221126-1+deb12u1_amd64.deb ...
Unpacking iputils-ping (3:20221126-1+deb12u1) over (3:20221126-1) ...
Preparing to unpack .../03-vim-tiny_2%3a9.0.1378-2+deb12u2_amd64.deb ...
Unpacking vim-tiny (2:9.0.1378-2+deb12u2) over (2:9.0.1378-2) ...
 ⠼Preparing to unpack .../04-vim-common_2%3a9.0.1378-2+deb12u2_all.deb ...
Unpacking vim-common (2:9.0.1378-2+deb12u2) over (2:9.0.1378-2) ...
Preparing to unpack .../05-libnghttp2-14_1.52.0-1+deb12u2_amd64.deb ...
Unpacking libnghttp2-14:amd64 (1.52.0-1+deb12u2) over (1.52.0-1+deb12u1) ...
Preparing to unpack .../06-bind9-dnsutils_1%3a9.18.33-1~deb12u2_amd64.deb ...
Unpacking bind9-dnsutils (1:9.18.33-1~deb12u2) over (1:9.18.28-1~deb12u2) ...
 ⠴Preparing to unpack .../07-bind9-host_1%3a9.18.33-1~deb12u2_amd64.deb ...
Unpacking bind9-host (1:9.18.33-1~deb12u2) over (1:9.18.28-1~deb12u2) ...
Preparing to unpack .../08-bind9-libs_1%3a9.18.33-1~deb12u2_amd64.deb ...
Unpacking bind9-libs:amd64 (1:9.18.33-1~deb12u2) over (1:9.18.28-1~deb12u2) ...
 ⠦Preparing to unpack .../09-libc-l10n_2.36-9+deb12u10_all.deb ...
Unpacking libc-l10n (2.36-9+deb12u10) over (2.36-9+deb12u8) ...
 ⠧Preparing to unpack .../10-locales_2.36-9+deb12u10_all.deb ...
Unpacking locales (2.36-9+deb12u10) over (2.36-9+deb12u8) ...
 ⠙Preparing to unpack .../11-wget_1.21.3-1+deb12u1_amd64.deb ...
Unpacking wget (1.21.3-1+deb12u1) over (1.21.3-1+b2) ...
 ⠹Preparing to unpack .../12-xz-utils_5.4.1-1_amd64.deb ...
Unpacking xz-utils (5.4.1-1) over (5.4.1-0.2) ...
 ⠸Preparing to unpack .../13-bsdextrautils_2.38.1-5+deb12u3_amd64.deb ...
Unpacking bsdextrautils (2.38.1-5+deb12u3) over (2.38.1-5+deb12u1) ...
Preparing to unpack .../14-distro-info-data_0.58+deb12u3_all.deb ...
Unpacking distro-info-data (0.58+deb12u3) over (0.58+deb12u2) ...
 ⠼Preparing to unpack .../15-libcurl3-gnutls_7.88.1-10+deb12u12_amd64.deb ...
Unpacking libcurl3-gnutls:amd64 (7.88.1-10+deb12u12) over (7.88.1-10+deb12u7) ...
Preparing to unpack .../16-openssl_3.0.15-1~deb12u1_amd64.deb ...
Unpacking openssl (3.0.15-1~deb12u1) over (3.0.14-1~deb12u2) ...
 ⠴Preparing to unpack .../17-python3-pkg-resources_66.1.1-1+deb12u1_all.deb ...
 ⠦Unpacking python3-pkg-resources (66.1.1-1+deb12u1) over (66.1.1-1) ...
 ⠧Preparing to unpack .../18-python3-urllib3_1.26.12-1+deb12u1_all.deb ...
Unpacking python3-urllib3 (1.26.12-1+deb12u1) over (1.26.12-1) ...
 ⠇Preparing to unpack .../19-ssh_1%3a9.2p1-2+deb12u5_all.deb ...
Unpacking ssh (1:9.2p1-2+deb12u5) over (1:9.2p1-2+deb12u3) ...
Setting up python3-pkg-resources (66.1.1-1+deb12u1) ...
 ⠏Setting up libexpat1:amd64 (2.5.0-1+deb12u1) ...
Setting up systemd-sysv (252.36-1~deb12u1) ...
Setting up libc-l10n (2.36-9+deb12u10) ...
 ⠋Setting up bsdextrautils (2.38.1-5+deb12u3) ...
Setting up wget (1.21.3-1+deb12u1) ...
Setting up distro-info-data (0.58+deb12u3) ...
Setting up openssh-client (1:9.2p1-2+deb12u5) ...
Setting up libsqlite3-0:amd64 (3.40.1-2+deb12u1) ...
Setting up libnghttp2-14:amd64 (1.52.0-1+deb12u2) ...
Setting up libnss-systemd:amd64 (252.36-1~deb12u1) ...
Setting up libcurl3-gnutls:amd64 (7.88.1-10+deb12u12) ...
Setting up locales (2.36-9+deb12u10) ...
 ⠹Generating locales (this might take a while)...
 ⠏ done.UTF-8...
Generation complete.
Setting up tzdata (2025b-0+deb12u1) ...
 ⠋
Current default time zone: 'Asia/Kolkata'
Local time is now:      Mon Apr 14 05:57:10 IST 2025.
Universal Time is now:  Mon Apr 14 00:27:10 UTC 2025.
Run 'dpkg-reconfigure tzdata' if you wish to change it.

Setting up perl-modules-5.36 (5.36.0-7+deb12u2) ...
Setting up vim-common (2:9.0.1378-2+deb12u2) ...
Setting up xz-utils (5.4.1-1) ...
Setting up systemd-timesyncd (252.36-1~deb12u1) ...
 ⠹Setting up udev (252.36-1~deb12u1) ...
 ⠦Setting up ucf (3.0043+nmu1+deb12u1) ...
Setting up python3-urllib3 (1.26.12-1+deb12u1) ...
 ⠧Setting up libfdisk1:amd64 (2.38.1-5+deb12u3) ...
Setting up libpam-systemd:amd64 (252.36-1~deb12u1) ...
 ⠇Setting up libpython3.11-minimal:amd64 (3.11.2-6+deb12u5) ...
Setting up openssl (3.0.15-1~deb12u1) ...
Setting up iputils-ping (3:20221126-1+deb12u1) ...
Setting up libperl5.36:amd64 (5.36.0-7+deb12u2) ...
Setting up python3.11-minimal (3.11.2-6+deb12u5) ...
 ⠙Setting up openssh-sftp-server (1:9.2p1-2+deb12u5) ...
Setting up openssh-server (1:9.2p1-2+deb12u5) ...
 ⠼rescue-ssh.target is a disabled or a static unit not running, not starting it.
 ⠦Could not execute systemctl:  at /usr/bin/deb-systemd-invoke line 145.
Setting up bind9-libs:amd64 (1:9.18.33-1~deb12u2) ...
Setting up libpython3.11-stdlib:amd64 (3.11.2-6+deb12u5) ...
Setting up vim-tiny (2:9.0.1378-2+deb12u2) ...
Setting up fdisk (2.38.1-5+deb12u3) ...
Setting up perl (5.36.0-7+deb12u2) ...
Setting up bind9-host (1:9.18.33-1~deb12u2) ...
Setting up python3.11 (3.11.2-6+deb12u5) ...
 ⠋Setting up ssh (1:9.2p1-2+deb12u5) ...
Setting up bind9-dnsutils (1:9.18.33-1~deb12u2) ...
Processing triggers for libc-bin (2.36-9+deb12u10) ...
Processing triggers for systemd (252.36-1~deb12u1) ...
Processing triggers for man-db (2.11.2-2) ...
 ⠼Processing triggers for dbus (1.14.10-1~deb12u1) ...
Processing triggers for mailcap (3.70+nmu1) ...
  ✔️  Updated Container OS
Hit:1 http://deb.debian.org/debian bookworm InRelease
Hit:2 http://deb.debian.org/debian bookworm-updates InRelease
Hit:3 http://security.debian.org bookworm-security InRelease
Reading package lists... Done
Reading package lists... Done
Building dependency tree... Done
Reading state information... Done
The following additional packages will be installed:
  libcurl4 libglib2.0-0 libglib2.0-data libgpm2 mc-data shared-mime-info unzip xdg-user-dirs
Suggested packages:
  low-memory-monitor gpm arj catdvi | texlive-binaries dbview djvulibre-bin epub-utils genisoimage gv imagemagick libaspell-dev links | w3m | lynx odt2txt
  poppler-utils python python-boto python-tz unar wimtools xpdf | pdf-viewer zip
 ⠧The following NEW packages will be installed:
  curl libcurl4 libglib2.0-0 libglib2.0-data libgpm2 mc mc-data shared-mime-info sudo unzip xdg-user-dirs
0 upgraded, 11 newly installed, 0 to remove and 0 not upgraded.
Need to get 8,040 kB of archives.
After this operation, 35.3 MB of additional disk space will be used.
Get:1 http://deb.debian.org/debian bookworm/main amd64 sudo amd64 1.9.13p3-1+deb12u1 [1,889 kB]
Get:2 http://deb.debian.org/debian bookworm/main amd64 libcurl4 amd64 7.88.1-10+deb12u12 [391 kB]
Get:3 http://deb.debian.org/debian bookworm/main amd64 curl amd64 7.88.1-10+deb12u12 [315 kB]
Get:4 http://deb.debian.org/debian bookworm/main amd64 libglib2.0-0 amd64 2.74.6-2+deb12u5 [1,403 kB]
Get:5 http://deb.debian.org/debian bookworm/main amd64 libglib2.0-data all 2.74.6-2+deb12u5 [1,209 kB]
Get:6 http://deb.debian.org/debian bookworm/main amd64 libgpm2 amd64 1.20.7-10+b1 [14.2 kB]
Get:7 http://deb.debian.org/debian bookworm/main amd64 mc-data all 3:4.8.29-2 [1,346 kB]
Get:8 http://deb.debian.org/debian bookworm/main amd64 mc amd64 3:4.8.29-2 [522 kB]
Get:9 http://deb.debian.org/debian bookworm/main amd64 shared-mime-info amd64 2.2-1 [729 kB]
Get:10 http://deb.debian.org/debian bookworm/main amd64 unzip amd64 6.0-28 [166 kB]
Get:11 http://deb.debian.org/debian bookworm/main amd64 xdg-user-dirs amd64 0.18-1 [54.4 kB]
Fetched 8,040 kB in 0s (63.0 MB/s)       
 ⠋Selecting previously unselected package sudo.
(Reading database ... 19136 files and directories currently installed.)
Preparing to unpack .../00-sudo_1.9.13p3-1+deb12u1_amd64.deb ...
Unpacking sudo (1.9.13p3-1+deb12u1) ...
 ⠙Selecting previously unselected package libcurl4:amd64.
Preparing to unpack .../01-libcurl4_7.88.1-10+deb12u12_amd64.deb ...
Unpacking libcurl4:amd64 (7.88.1-10+deb12u12) ...
Selecting previously unselected package curl.
Preparing to unpack .../02-curl_7.88.1-10+deb12u12_amd64.deb ...
Unpacking curl (7.88.1-10+deb12u12) ...
 ⠹Selecting previously unselected package libglib2.0-0:amd64.
Preparing to unpack .../03-libglib2.0-0_2.74.6-2+deb12u5_amd64.deb ...
Unpacking libglib2.0-0:amd64 (2.74.6-2+deb12u5) ...
 ⠸Selecting previously unselected package libglib2.0-data.
Preparing to unpack .../04-libglib2.0-data_2.74.6-2+deb12u5_all.deb ...
Unpacking libglib2.0-data (2.74.6-2+deb12u5) ...
 ⠼Selecting previously unselected package libgpm2:amd64.
Preparing to unpack .../05-libgpm2_1.20.7-10+b1_amd64.deb ...
Unpacking libgpm2:amd64 (1.20.7-10+b1) ...
Selecting previously unselected package mc-data.
Preparing to unpack .../06-mc-data_3%3a4.8.29-2_all.deb ...
Unpacking mc-data (3:4.8.29-2) ...
 ⠦Selecting previously unselected package mc.
Preparing to unpack .../07-mc_3%3a4.8.29-2_amd64.deb ...
Unpacking mc (3:4.8.29-2) ...
 ⠧Selecting previously unselected package shared-mime-info.
Preparing to unpack .../08-shared-mime-info_2.2-1_amd64.deb ...
Unpacking shared-mime-info (2.2-1) ...
 ⠇Selecting previously unselected package unzip.
Preparing to unpack .../09-unzip_6.0-28_amd64.deb ...
Unpacking unzip (6.0-28) ...
Selecting previously unselected package xdg-user-dirs.
Preparing to unpack .../10-xdg-user-dirs_0.18-1_amd64.deb ...
Unpacking xdg-user-dirs (0.18-1) ...
 ⠏Setting up libgpm2:amd64 (1.20.7-10+b1) ...
Setting up xdg-user-dirs (0.18-1) ...
Setting up libglib2.0-0:amd64 (2.74.6-2+deb12u5) ...
No schema files found: doing nothing.
Setting up unzip (6.0-28) ...
Setting up libglib2.0-data (2.74.6-2+deb12u5) ...
Setting up shared-mime-info (2.2-1) ...
 ⠸Setting up sudo (1.9.13p3-1+deb12u1) ...
 ⠼Setting up mc-data (3:4.8.29-2) ...
Setting up libcurl4:amd64 (7.88.1-10+deb12u12) ...
Setting up curl (7.88.1-10+deb12u12) ...
Setting up mc (3:4.8.29-2) ...
update-alternatives: using /usr/bin/mcview to provide /usr/bin/view (view) in auto mode
Processing triggers for man-db (2.11.2-2) ...
 ⠇Processing triggers for mailcap (3.70+nmu1) ...
Processing triggers for libc-bin (2.36-9+deb12u10) ...
  ✔️  Core dependencies installed
Reading package lists... Done
Building dependency tree... Done
Reading state information... Done
The following additional packages will be installed:
  dirmngr gnupg gnupg-l10n gnupg-utils gpg-agent gpg-wks-client gpg-wks-server gpgconf gpgsm libassuan0 libksba8 libnpth0 pinentry-curses
Suggested packages:
  dbus-user-session pinentry-gnome3 tor parcimonie xloadimage scdaemon pinentry-doc
The following NEW packages will be installed:
  dirmngr gnupg gnupg-l10n gnupg-utils gpg gpg-agent gpg-wks-client gpg-wks-server gpgconf gpgsm libassuan0 libksba8 libnpth0 pinentry-curses
0 upgraded, 14 newly installed, 0 to remove and 0 not upgraded.
Need to get 7,881 kB of archives.
After this operation, 16.0 MB of additional disk space will be used.
Get:1 http://deb.debian.org/debian bookworm/main amd64 libassuan0 amd64 2.5.5-5 [48.5 kB]
Get:2 http://deb.debian.org/debian bookworm/main amd64 gpgconf amd64 2.2.40-1.1 [564 kB]
Get:3 http://deb.debian.org/debian bookworm/main amd64 libksba8 amd64 1.6.3-2 [128 kB]
Get:4 http://deb.debian.org/debian bookworm/main amd64 libnpth0 amd64 1.6-3 [19.0 kB]
Get:5 http://deb.debian.org/debian bookworm/main amd64 dirmngr amd64 2.2.40-1.1 [792 kB]
Get:6 http://deb.debian.org/debian bookworm/main amd64 gnupg-l10n all 2.2.40-1.1 [1,093 kB]
Get:7 http://deb.debian.org/debian bookworm/main amd64 gnupg-utils amd64 2.2.40-1.1 [927 kB]
Get:8 http://deb.debian.org/debian bookworm/main amd64 gpg amd64 2.2.40-1.1 [949 kB]
Get:9 http://deb.debian.org/debian bookworm/main amd64 pinentry-curses amd64 1.2.1-1 [77.4 kB]
Get:10 http://deb.debian.org/debian bookworm/main amd64 gpg-agent amd64 2.2.40-1.1 [695 kB]
Get:11 http://deb.debian.org/debian bookworm/main amd64 gpg-wks-client amd64 2.2.40-1.1 [541 kB]
Get:12 http://deb.debian.org/debian bookworm/main amd64 gpg-wks-server amd64 2.2.40-1.1 [531 kB]
Get:13 http://deb.debian.org/debian bookworm/main amd64 gpgsm amd64 2.2.40-1.1 [671 kB]
Get:14 http://deb.debian.org/debian bookworm/main amd64 gnupg all 2.2.40-1.1 [846 kB]
Fetched 7,881 kB in 0s (56.3 MB/s)
 ⠼Selecting previously unselected package libassuan0:amd64.
(Reading database ... 20056 files and directories currently installed.)
Preparing to unpack .../00-libassuan0_2.5.5-5_amd64.deb ...
Unpacking libassuan0:amd64 (2.5.5-5) ...
Selecting previously unselected package gpgconf.
Preparing to unpack .../01-gpgconf_2.2.40-1.1_amd64.deb ...
Unpacking gpgconf (2.2.40-1.1) ...
Selecting previously unselected package libksba8:amd64.
Preparing to unpack .../02-libksba8_1.6.3-2_amd64.deb ...
Unpacking libksba8:amd64 (1.6.3-2) ...
Selecting previously unselected package libnpth0:amd64.
Preparing to unpack .../03-libnpth0_1.6-3_amd64.deb ...
Unpacking libnpth0:amd64 (1.6-3) ...
Selecting previously unselected package dirmngr.
Preparing to unpack .../04-dirmngr_2.2.40-1.1_amd64.deb ...
Unpacking dirmngr (2.2.40-1.1) ...
 ⠦Selecting previously unselected package gnupg-l10n.
Preparing to unpack .../05-gnupg-l10n_2.2.40-1.1_all.deb ...
Unpacking gnupg-l10n (2.2.40-1.1) ...
 ⠧Selecting previously unselected package gnupg-utils.
Preparing to unpack .../06-gnupg-utils_2.2.40-1.1_amd64.deb ...
Unpacking gnupg-utils (2.2.40-1.1) ...
Selecting previously unselected package gpg.
Preparing to unpack .../07-gpg_2.2.40-1.1_amd64.deb ...
Unpacking gpg (2.2.40-1.1) ...
 ⠇Selecting previously unselected package pinentry-curses.
Preparing to unpack .../08-pinentry-curses_1.2.1-1_amd64.deb ...
Unpacking pinentry-curses (1.2.1-1) ...
Selecting previously unselected package gpg-agent.
Preparing to unpack .../09-gpg-agent_2.2.40-1.1_amd64.deb ...
Unpacking gpg-agent (2.2.40-1.1) ...
Selecting previously unselected package gpg-wks-client.
Preparing to unpack .../10-gpg-wks-client_2.2.40-1.1_amd64.deb ...
Unpacking gpg-wks-client (2.2.40-1.1) ...
Selecting previously unselected package gpg-wks-server.
Preparing to unpack .../11-gpg-wks-server_2.2.40-1.1_amd64.deb ...
Unpacking gpg-wks-server (2.2.40-1.1) ...
 ⠏Selecting previously unselected package gpgsm.
Preparing to unpack .../12-gpgsm_2.2.40-1.1_amd64.deb ...
Unpacking gpgsm (2.2.40-1.1) ...
Selecting previously unselected package gnupg.
Preparing to unpack .../13-gnupg_2.2.40-1.1_all.deb ...
Unpacking gnupg (2.2.40-1.1) ...
 ⠋Setting up libksba8:amd64 (1.6.3-2) ...
Setting up libnpth0:amd64 (1.6-3) ...
Setting up libassuan0:amd64 (2.5.5-5) ...
Setting up gnupg-l10n (2.2.40-1.1) ...
Setting up gpgconf (2.2.40-1.1) ...
Setting up gpg (2.2.40-1.1) ...
Setting up gnupg-utils (2.2.40-1.1) ...
Setting up pinentry-curses (1.2.1-1) ...
Setting up gpg-agent (2.2.40-1.1) ...
Created symlink /etc/systemd/user/sockets.target.wants/gpg-agent-browser.socket → /usr/lib/systemd/user/gpg-agent-browser.socket.
 ⠙Created symlink /etc/systemd/user/sockets.target.wants/gpg-agent-extra.socket → /usr/lib/systemd/user/gpg-agent-extra.socket.
Created symlink /etc/systemd/user/sockets.target.wants/gpg-agent-ssh.socket → /usr/lib/systemd/user/gpg-agent-ssh.socket.
 ⠹Created symlink /etc/systemd/user/sockets.target.wants/gpg-agent.socket → /usr/lib/systemd/user/gpg-agent.socket.
Setting up gpgsm (2.2.40-1.1) ...
Setting up dirmngr (2.2.40-1.1) ...
 ⠸Created symlink /etc/systemd/user/sockets.target.wants/dirmngr.socket → /usr/lib/systemd/user/dirmngr.socket.
Setting up gpg-wks-server (2.2.40-1.1) ...
Setting up gpg-wks-client (2.2.40-1.1) ...
Setting up gnupg (2.2.40-1.1) ...
Processing triggers for man-db (2.11.2-2) ...
 ⠴Processing triggers for libc-bin (2.36-9+deb12u10) ...
  ✔️  Installed Dependencies
Hit:1 http://deb.debian.org/debian bookworm InRelease
Hit:2 http://security.debian.org bookworm-security InRelease            
Hit:3 http://deb.debian.org/debian bookworm-updates InRelease           
Get:4 https://debian.neo4j.com stable InRelease [44.2 kB]               
Get:5 https://debian.neo4j.com stable/latest amd64 Packages [1,522 B]
Fetched 45.8 kB in 1s (48.3 kB/s)
Reading package lists... Done
Reading package lists... Done
Building dependency tree... Done
Reading state information... Done
Some packages could not be installed. This may mean that you have
requested an impossible situation or if you are using the unstable
distribution that some required packages have not yet been created
or been moved out of Incoming.
The following information may help to resolve the situation:

The following packages have unmet dependencies:
 neo4j : Depends: java21-runtime but it is not installable or
                  jdk-21 but it is not installable
         Depends: cypher-shell (>= 1:2025.03.0) but it is not going to be installed
         Depends: cypher-shell (< 2:0.0.0) but it is not going to be installed
E: Unable to correct problems, you have held broken packages.
 ⠏
[ERROR] in line 25: exit code 0: while executing command $STD apt-get install -y neo4j

### 🖼️ Additional context (optional).

_No response_

--- a/ct/neo4j.sh
+++ b/ct/neo4j.sh
@@ -20,18 +20,32 @@ color
 catch_errors
 
 function update_script() {
-    header_info
-    check_container_storage
-    check_container_resources
-    if [[ ! -d /etc/neo4j ]]; then
-        msg_error "No ${APP} Installation Found!"
-        exit
-    fi
-    msg_info "Updating ${APP}"
-    $STD apt-get update
-    $STD apt-get -y upgrade
-    msg_ok "Updated Successfully"
+  header_info
+  check_container_storage
+  check_container_resources
+  if [[ ! -d /etc/neo4j ]]; then
+    msg_error "No ${APP} Installation Found!"
     exit
+  fi
+  if ! dpkg -l | grep -q temurin-21-jre; then
+    msg_info "Installing Adoptium JDK"
+    $STD apt-get install -y \
+      gnupg2 \
+      lsb-release \
+      gpg \
+      apt-transport-https
+    mkdir -p /etc/apt/keyrings
+    curl -fsSL https://packages.adoptium.net/artifactory/api/gpg/key/public | gpg --dearmor >/etc/apt/trusted.gpg.d/adoptium.gpg
+    echo "deb https://packages.adoptium.net/artifactory/deb $(awk -F= '/^VERSION_CODENAME/{print$2}' /etc/os-release) main" >/etc/apt/sources.list.d/adoptium.list
+    $STD apt-get update
+    $STD apt-get install -y temurin-21-jre
+    msg_ok "Adoptium JDK installed"
+  fi
+  msg_info "Updating ${APP}"
+  $STD apt-get update
+  $STD apt-get -y upgrade
+  msg_ok "Updated Successfully"
+  exit
 }
 
 start

--- a/ct/neo4j.sh
+++ b/ct/neo4j.sh
@@ -31,9 +31,7 @@ function update_script() {
     msg_info "Installing Adoptium JDK"
     $STD apt-get install -y \
       gnupg2 \
-      lsb-release \
-      gpg \
-      apt-transport-https
+      lsb-release
     mkdir -p /etc/apt/keyrings
     curl -fsSL https://packages.adoptium.net/artifactory/api/gpg/key/public | gpg --dearmor >/etc/apt/trusted.gpg.d/adoptium.gpg
     echo "deb https://packages.adoptium.net/artifactory/deb $(awk -F= '/^VERSION_CODENAME/{print$2}' /etc/os-release) main" >/etc/apt/sources.list.d/adoptium.list

--- a/install/neo4j-install.sh
+++ b/install/neo4j-install.sh
@@ -17,10 +17,8 @@ update_os
 msg_info "Installing Dependencies"
 $STD apt-get install -y \
   gnupg2 \
-  lsb-release \
-  gpg \
-  apt-transport-https
-msg_ok "Installed Dependencies"
+  lsb-release 
+  msg_ok "Installed Dependencies"
 
 msg_info "Setting up Adoptium Repository"
 mkdir -p /etc/apt/keyrings

--- a/install/neo4j-install.sh
+++ b/install/neo4j-install.sh
@@ -18,6 +18,13 @@ msg_info "Installing Dependencies"
 $STD apt-get install -y gpg
 msg_ok "Installed Dependencies"
 
+msg_info "Installing OpenJDK"
+curl -fsSL https://packages.adoptium.net/artifactory/api/gpg/key/public | gpg --dearmor >/etc/apt/trusted.gpg.d/adoptium.gpg
+echo 'deb [arch=amd64 signed-by=/etc/apt/trusted.gpg.d/adoptium.gpg] https://packages.adoptium.net/artifactory/deb bookworm main' >/etc/apt/sources.list.d/adoptium.list
+$STD apt-get update
+$STD apt-get install -y temurin-21-jre
+msg_ok "Installed OpenJDK"
+
 msg_info "Installing Neo4j (patience)"
 curl -fsSL "https://debian.neo4j.com/neotechnology.gpg.key" | gpg --dearmor -o /etc/apt/keyrings/neotechnology.gpg
 echo 'deb [signed-by=/etc/apt/keyrings/neotechnology.gpg] https://debian.neo4j.com stable latest' >/etc/apt/sources.list.d/neo4j.list

--- a/install/neo4j-install.sh
+++ b/install/neo4j-install.sh
@@ -15,20 +15,24 @@ network_check
 update_os
 
 msg_info "Installing Dependencies"
-$STD apt-get install -y gpg
+$STD apt-get install -y \
+  gnupg2 \
+  lsb-release \
+  gpg \
+  apt-transport-https
 msg_ok "Installed Dependencies"
 
-msg_info "Installing OpenJDK"
-curl -fsSL https://packages.adoptium.net/artifactory/api/gpg/key/public | gpg --dearmor >/etc/apt/trusted.gpg.d/adoptium.gpg
-echo 'deb [arch=amd64 signed-by=/etc/apt/trusted.gpg.d/adoptium.gpg] https://packages.adoptium.net/artifactory/deb bookworm main' >/etc/apt/sources.list.d/adoptium.list
-$STD apt-get update
-$STD apt-get install -y temurin-21-jre
-msg_ok "Installed OpenJDK"
+msg_info "Setting up Adoptium Repository"
+mkdir -p /etc/apt/keyrings
+curl -fsSL "https://packages.adoptium.net/artifactory/api/gpg/key/public" | gpg --dearmor >/etc/apt/trusted.gpg.d/adoptium.gpg
+echo "deb https://packages.adoptium.net/artifactory/deb $(awk -F= '/^VERSION_CODENAME/{print$2}' /etc/os-release) main" >/etc/apt/sources.list.d/adoptium.list
+msg_ok "Set up Adoptium Repository"
 
 msg_info "Installing Neo4j (patience)"
 curl -fsSL "https://debian.neo4j.com/neotechnology.gpg.key" | gpg --dearmor -o /etc/apt/keyrings/neotechnology.gpg
 echo 'deb [signed-by=/etc/apt/keyrings/neotechnology.gpg] https://debian.neo4j.com stable latest' >/etc/apt/sources.list.d/neo4j.list
 $STD apt-get update
+$STD apt-get install -y temurin-21-jre
 $STD apt-get install -y neo4j
 sed -i '/server.default_listen_address/s/^#//' /etc/neo4j/neo4j.conf
 systemctl enable -q --now neo4j

--- a/install/neo4j-install.sh
+++ b/install/neo4j-install.sh
@@ -17,8 +17,8 @@ update_os
 msg_info "Installing Dependencies"
 $STD apt-get install -y \
   gnupg2 \
-  lsb-release 
-  msg_ok "Installed Dependencies"
+  lsb-release
+msg_ok "Installed Dependencies"
 
 msg_info "Setting up Adoptium Repository"
 mkdir -p /etc/apt/keyrings

